packages/backend/src/db/bill/bill.schema.ts

```diff
@@ -0,0 +1,46 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document, Types } from 'mongoose';
+
+export type BillDocument = Bill & Document;
+
+@Schema()
+export class Bill {
+  @Prop()
+  name: string;
+
+  @Prop()
+  description: string;
+
+  @Prop({ type: Types.ObjectId, ref: 'House' })
+  house: Types.ObjectId;
+
+  @Prop({ type: Types.ObjectId, ref: 'User' })
+  owner: Types.ObjectId;
+
+  @Prop()
+  due: Date;
+
+  @Prop({ type: Types.ObjectId, ref: 'User' })
+  users: {
+    id: Types.ObjectId;
```

Comment:
By the looks of the issue specification this should be the users firebaseId

Reply:
And therefore a string not a objectID

Reply:
The way I did it was do the conversion to firebase id when returning the DTO but store the objectid in the database for better consistency. This could be changed to just store the firebaseid, not sure which was is better though

Reply:
Given that we are not using the user id to populate the nested documents or anything like that then its probably not an issue that it is a firebaseID stored in the db. I think we should use the same reference for all these operations rather than converting between. 

An issue that I can see happening is if the frontend is passing firebaseId in the requestBody but we are expecting a documentID instead.

Reply:
Actually sorry I get what you mean now. It is probably okay to leave as is

Reply:
I also accounted for the possible issue by expecting all incoming ids to be firebaseIDs so I do the conversion to ObjectID each time. It does result in quite a few database calls though.

Reply:
Yeah I think if we had more time it would be better to change, but thats alright.